src/common/components/vanilla/heading/index.js

```diff
@@ -4,15 +4,17 @@ import styles from './heading.css';
 
 const Heading = (props) => {
   const H = props.heading || 'h1';
+  const A = props.align || 'left';
```

Comment:
I'd rather call it `align`.

`H` is used above only because it's used later as an `<H>` tag (and I didn't want invent another name as `Heading` couldn't be used because it's name of this component already), but for other variables it would use more readable names.